src/TestFramework/DataFaker.php

```diff
@@ -1111,6 +1111,16 @@ public function riskScore()
         return $this->intBetween(-2, 100);
     }
 
+    /**
+     * Return a subscription cancel reason
+     *
+     * @return string
+     */
+    public function subscriptionCancelReason()
```

Comment:
Should this be `subscriptionCancelReasonCode`?

Reply:
see above